discord/installer/install_linux.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+INSTALL_OPTIONS=("Download correct node and install" "Install using local version" "Exit")
+
+runInstaller() {
+    echo "Please enter the install directory";
+    read INSTALL_DIR;
+    echo "Installing to $INSTALL_DIR";
```

Comment:
We should ensure that we can find an `app.asar` file or some other similar assurance that this is actually the discord directory before we copy our files and attempt to install.